backend/messaging/wizard_bus.go

```diff
@@ -87,11 +85,10 @@ func (b *WizardBus) Start() error {
 func (b *WizardBus) Stop() error {
 	b.running.Store(false)
 	close(b.errchan)
-	b.topicsMu.Lock()
-	for _, wTopic := range b.topics {
-		wTopic.Close()
-	}
-	b.topicsMu.Unlock()
+	b.topics.Range(func(_, value interface{}) bool {
+		value.(*wizardTopic).Close()
```

Comment:
Everywhere else you use a type assertion, shouldn't you check that the assertion is ok? 

For all we know `value` is null or is not even of type `*wizardTopic`. That would probably be a programming error somewhere else in this case, but this check used to be mostly done automatically via the type of `topics` (`map[string]*WizardTopic`). Now `topics` is essentially of type `map[interface{}]interface{}`.

Reply:
Regarding this being a patch:

It doesn't require a minor version bump, but I didn't want to interrupt release goings-on at the time. I could target the 6.0 release branch.

Regarding the type assertion, I believe it's desirable to get a crash here if the data type isn't `*wizardTopic`. There is no other valid datatype that can be stored in the map.

Reply:
I only partially agree: in my opinion, it's desirable to get a crash in those cases _if and only if_ we are sure to always catch these potential crashes before releasing the software. If we miss cases during our various stages of testing, we could end up releasing software that can crash in production rather than gracefully handle the unexpected.

The type system used to enforce the type of the values in the map at compile time, so we were sure to catch a big class of errors. Now it can't do that for us anymore. That's why I'm in favour of checking the result of the assertion.

If you're confident we have enough coverage to always to skip that check, ship it!

Reply:
I see your point, but I think in this case it would be better if the software just didn't function at all, rather than be partially broken. At least then it's obvious to operators! :)

I'll double-check that the code is well-covered by unit tests.